.circleci/config.yml

```diff
@@ -34,12 +42,19 @@ executors:
 
 jobs:
   validate:
-    executor:
-      name: hmpps/java
-      tag: "16.0"
-    docker: *db_docker_config
+    docker: *db_localstack_docker_config
     steps:
       - checkout
+      - hmpps/install_aws_cli
+      - hmpps/wait_till_ready:
+          port: 4566
+      - run:
+          name: Create S3 bucket
+          command: |
+            export AWS_ACCESS_KEY_ID=dontcare
+            export AWS_SECRET_ACCESS_KEY=dontcare
+            export AWS_DEFAULT_REGION=eu-west-2
+            aws s3api create-bucket --bucket test-manage-recalls-api --region eu-west-2 --endpoint-url http://localhost:4566
```

Comment:
minor, but not sure you need region here if you have the env var

Reply:
ok will try without